tests_new/integration_tests/modules/s3_datasets/test_s3_folders.py

```diff
@@ -49,9 +49,8 @@ def test_get_folder(client1, folders_fixture_name, request):
 def test_get_folder_unauthorized(client2, folders_fixture_name, request):
     folders = request.getfixturevalue(folders_fixture_name)
     folder = folders[0]
-    assert_that(get_folder).raises(GqlError).when_called_with(client2, locationUri=folder.locationUri).contains(
-        'UnauthorizedOperation', 'GET_DATASET_FOLDER', folder.locationUri
-    )
+    to_be_error = get_folder(client2, locationUri=folder.locationUri)
+    assert_that(to_be_error).contains('UnauthorizedOperation', 'GET_DATASET_FOLDER', folder.locationUri)
```

Comment:
you mean using `assert_that(function).raises(GqlError).when_called_with(args)`? I think it would be good to keep the standard

Reply:
[get_folder](https://github.com/data-dot-all/dataall/blob/main/tests_new/integration_tests/modules/s3_datasets/queries.py#L378) doesn't return the `error`  so I doubt `to_be_error` will contain this information.

Reply:
Hm. Strange, actually. Adriana and me decided to remove this test for good, since the backend has no access resrtictions. It will never work. I'll remove it